panel/src/components/Forms/Input/SlugInput.vue

```diff
@@ -17,7 +17,7 @@
 		spellcheck="false"
 		type="text"
 		class="k-text-input"
-		v-on="listeners"
+		@input="$emit('input', $event)"
```

Comment:
Not sure about this one. Wouldn't this remove the validate listener from the text input, which could be a breaking change?

Reply:
Tru probably needs `@invalid="($invalid, $v) => $emit('invalid', $invalid, $v)"` as the other inputs as well

Reply:
Oh not, cause it's a native `input` element